src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -72,10 +73,10 @@ public static Address parseAddress(String address) throws ParseException {
     public static Tag parseTag(String tag) throws ParseException {
         requireNonNull(tag);
         String trimmedTag = tag.trim();
-        if (!Tag.isValidTagName(trimmedTag)) {
+        if (!Tag.isValidTag(trimmedTag)) {
```

Comment:
Good that you wanted to maintain consistency in how the isValid\[attribute\]() method is named across classes